lib/cli/src/commands/package/tag.rs

```diff
@@ -200,6 +200,18 @@ impl PackageTag {
             None
         };
 
+        let maybe_readme_content = match maybe_readme {
+            Some(readme) => {
+                let readme_path = Path::new(&readme);
```

Comment:
This path has to be relative to where the manifest lives. I don't think we are accounting for that on this code